mondey_backend/src/mondey_backend/routers/users.py

```diff
@@ -117,6 +119,22 @@ async def upload_child_image(
         write_file(file, filename)
         return {"ok": True}
 
+    @router.delete("/children-images/{child_id}")
+    async def delete_child_image(
+        session: SessionDep, current_active_user: CurrentActiveUserDep, child_id: int
+    ):
+        child = get_db_child(session, current_active_user, child_id)
+        image_path = pathlib.Path(
+            f"{app_settings.PRIVATE_FILES_PATH}/children/{child.id}.jpg"
+        )
+        if not image_path.exists():
+            raise HTTPException(404, detail="Image not found")
+        else:
+            child.has_image = False
+            session.commit()
+            image_path.unlink()
```

Comment:
I'd suggest not checking if the file exists, and just doing the else part of this, with .unlink(missing_ok=True) - if the file is already deleted for whatever reason then that's actually fine, and we shouldn't abort setting child.has_image to false imo